check_early_hints.js

```
@@ -0,0 +1,111 @@
+            console.log('❌ No Early Hints debug comments found in HTML');
+            console.log('This might mean:');
+            console.log('  - Early hints are not enabled');
+            console.log('  - The deployment is not running the latest code');
+            console.log('  - The page needs to be reloaded');
+          }
+
+          // Also check for Link headers with preload
+          const linkMatches = html.match(/<link[^>]*rel=["']preload["'][^>]*>/g);
+          if (linkMatches) {
+            console.log(`\n📦 Found ${linkMatches.length} preload links in HTML head:`);
+            linkMatches.slice(0, 5).forEach(link => {
```

ESLint

[error] Replace `link` with `(link)`

(prettier/prettier)